package_linter.py

```diff
@@ -1576,7 +1576,7 @@ def bind_public_ip(self):
                     return
 
                 for number, line in enumerate(content.split("\n"), 1):
-                    comment = ("#", "//", ";")
+                    comment = ("#", "//", ";", "/**", "*")
```

Comment:
(FYI, after discussion in DM, "/**" still needs to be replaced by "/*" and it will be good for me)

Reply:
```suggestion
                    comment = ("#", "//", ";", "/*", "*")
```